Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -93,10 +78,7 @@ public static AssertionScope Current
             private set => current = value;
         }
 
-        /// <summary>
-        /// Indicates that every argument passed into <see cref="FailWith"/> is displayed on a separate line.
-        /// </summary>
-        public AssertionScope UsingLineBreaks
+        public IAssertionScope UsingLineBreaks
```

Comment:
I'm not sure. The members of `IAssertionScope` are only used in chaining scenarios. I've included a couple of them, but not all. Any strong opinions?

Reply:
I haven't tried to invoke `AssertionScope`-only members myself, I just looked for changes that could break the public API.

Reply:
Normally, you're directly using the `AssertionScope` in a `using` block. The interface is only used for those complex chaining purposes. In the last commit, I've added almost all of them to the interface, just to be sure.

Reply:
But no, I don't have any strong opinions, except trying to avoid breaking changes.

Just as a note: the remaining differences between `AssertionScope` and `IAssertionScope` are:

```c#
public string Context { get; set; }
public void AddPreFormattedFailure(string formattedFailureMessage);
void AddNonReportable(string key, object value);
void AddReportable(string key, string value);
T Get<T>(string key);
```

Reply:
So? Approved?